GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java

```diff
@@ -407,33 +430,281 @@ public void loaded(GVRTexture texture, GVRAndroidResource ignored)
     }
 
     protected GVRContext mContext;
-    public GVRAssetLoader(GVRContext context) {
+    protected static final ResourceCache<GVRTexture> mTextureCache = new ResourceCache<GVRTexture>();
```

Comment:
Please look at the use of GVRContext.addResetOnRestartHandler to recreate the instance. Ideally we would not have static state but since we do, then you need to reset the instance.

The problem is that once the app is backgrounded, the process usually is not killed. If the app is started again the cache will stick around but there will be a new gl context created - the cache contents will be invalid and there will be cache hits.